### PR TITLE
New Top Nav

### PR DESCRIPTION
**Pull request by alex-kovoy**
This PR:

Moves Roles/Auth.Connectors/TrustedClusters from the cluster view to the Dashboard Top Nav.

Adds "People" and "Settings" screens. Both have their own features.

### CODE REVIEW

File: packages/teleport/src/components/useResources.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { useState } from 'shared/hooks';
```
Comment (kimlisa) [inline packages/teleport/src/components/useResources.ts:1]: missing license

File: packages/teleport/src/components/ResourceEditor/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import ResourceEditor from './ResourceEditor';
+export default ResourceEditor;
```
Comment (kimlisa) [inline packages/teleport/src/components/ResourceEditor/index.js:18]: no new line. That's weird, formatter should auto insert newline on save.

File: packages/teleport/src/dashboard/People/Roles/templates/role.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    logins: ['guest']
+
+  options:
+      # Limits user credentials to 8 hours. After the time to live (TTL) expires,
+      # users must re-login
+      max_session_ttl: 8h0m0s
+  # the deny section uses the identical format as the 'allow' section.
+  # the deny rules always override allow rules.
+  deny: {}
+version: v3
```
Comment (kimlisa) [inline packages/teleport/src/dashboard/People/Roles/templates/role.yaml:35]: newline

File: packages/teleport/src/dashboard/Settings/features/index.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//import getFeatures, { Feature } from './getFeatures';
+
+//export * from './getFeatures';
+
+//export { getFeatures, Feature };
```
Comment (kimlisa) [inline packages/teleport/src/dashboard/Settings/features/index.ts:21]: maybe this file not needed?

File: packages/teleport/src/dashboard/features/featurePeople.ts [hunk 5]
```diff
@@ -14,36 +14,36 @@ See the License for the specific language governing permissions and
...
-  Component = withFeature(this)(Account);
+class FeaturePeople extends FeatureBase {
+  Component = People;
 
   getRoute() {
     return {
-      title: 'Account',
-      path: cfg.routes.account,
-      exact: true,
+      title: 'People',
```
Comment (kimlisa) [inline packages/teleport/src/dashboard/features/featurePeople.ts:27]: `People` is ambiguous. I think `Team` is a better word here

Comment (alex-kovoy) [reply packages/teleport/src/dashboard/features/featurePeople.ts:27]: > People is ambiguous. I think Team is a better word here

we do not have teams but we do have people so it's better to be a bit ambiguous than sending a wrong message.

**Pull request merged**